src/util/tocFormatter.ts

```diff
@@ -0,0 +1,6 @@
+// See related issue: https://github.com/gatsbyjs/gatsby/issues/8982
+export function fixTocCodeTag(tocLinks: string) {
```

Comment:
Default param value w. empty string could be used to guard against malicious callers like `fixTocCodeTag(undefined);` or `fixTocCodeTag(null);`

<img width="509" alt="Screen Shot 2019-04-09 at 10 41 04 PM" src="https://user-images.githubusercontent.com/6441326/55847766-9776d180-5b18-11e9-9728-7b9044eb9f00.png">


```suggestion
export function fixTocCodeTag(tocLinks: string = '' ) {
```